apps/docs/src/app/platform/component-docs/platform-forms/input-group/platform-input-group-examples/platform-input-group-compact-example.component.html

```diff
@@ -1,10 +1,15 @@
+<label fd-form-label for="input-group-compact" id="input-group-compact-label">Compact</label>
 <fdp-input-group
+    [id]="'input-group-compact'"
```

Comment:
```suggestion
    id="input-group-compact"
```

Reply:
@olshansky I would love to do it, but it doesn't work